metadata-db-app/src/cmr/metadata_db/data/oracle/collection_table.clj

```diff
@@ -1,7 +1,8 @@
 (ns cmr.metadata-db.data.oracle.collection-table
   "Contains helper functions to create collection table."
   (:require
-   [clojure.java.jdbc :as j]))
+   [clojure.java.jdbc :as j]
```

Comment:
single letter imports is now discouraged by the project.

Reply:
done